slam_toolbox_ws/src/slam_simulation/launch/slam_sim.launch.py

```diff
@@ -66,10 +66,42 @@ def generate_launch_description():
         PythonLaunchDescriptionSource(gazebo_path)
     )
     
+    # TF Ref: 
+    # https://docs.ros.org/en/humble/Tutorials/Intermediate/Tf2/Writing-A-Tf2-Static-Broadcaster-Py.html#the-proper-way-to-publish-static-transforms
+    # Static TF : base_footprint -> base_link
+    foot_to_link_tf2 = Node(
+        package = "tf2_ros",
+        executable = "static_transform_publisher",
+        name = "foot_to_link",
+        arguments = ["0", "0", "0", "0", "0", "0", "base_footprint", "base_link"],
+        parameters=[{'use_sim_time': True}]
+    )
+    
+    # Static TF : base_link -> base_scan
+    link_to_scan_tf2 = Node(
+        package = "tf2_ros",
+        executable = "static_transform_publisher",
+        name = "link_to_scan",
+        arguments = ["0", "0", "0", "0", "0", "0", "base_link", "base_scan"],
+        parameters=[{'use_sim_time': True}]
+    )
+    
+    # Static TF : base_link -> imu
+    link_to_imu_tf2 = Node(
+        package = "tf2_ros",
+        executable = "static_transform_publisher",
+        name = "link_to_imu",
+        arguments = ["0", "0", "0", "0", "0", "0", "base_link", "imu_link"],
+        parameters=[{'use_sim_time': True}]
+    )
```

Comment:
It would be better to mention why these 3 links are necessary. Take `base_footprint -> base_link` as an example, it may be necessary since `base_footprint` is the base frame as set here: https://github.com/SteveMacenski/slam_toolbox/blob/8365b77710df4759cd96a14be2b3082ede22f15a/config/mapper_params_online_async.yaml#L15